DigitalOffice/Tests/CompanyServiceTests/Commands/GetPositionsListCommandTests.cs

```diff
@@ -0,0 +1,35 @@
+﻿using LT.DigitalOffice.CompanyService.Commands;
+using LT.DigitalOffice.CompanyService.Commands.Interfaces;
+using LT.DigitalOffice.CompanyService.Models;
+using LT.DigitalOffice.CompanyService.Repositories.Interfaces;
+using Moq;
+using NUnit.Framework;
+using System.Collections.Generic;
+
+namespace LT.DigitalOffice.CompanyServiceUnitTests.Commands
+{
+    public class GetRightsListCommandTests
+    {
+        private Mock<ICompanyRepository> repositoryMock;
+        private IGetPositionsListCommand command;
+
+        [SetUp]
+        public void SetUp()
+        {
+            repositoryMock = new Mock<ICompanyRepository>();
+            command = new GetPositionsListCommand(repositoryMock.Object);
+            var position = new Position { Name = "Position", Description = "Description" };
+            var positionsList = new List<Position> { position };
+            repositoryMock.Setup(repository => repository.GetPositionsList()).Returns(positionsList);
+        }
+
+        [Test]
+        public void GetPositionsListCommandSuccess()
+        {
+            var result = command.Execute();
+
+            Assert.DoesNotThrow(() => command.Execute());
+            Assert.IsNotNull(result);
```

Comment:
why not compare with the expected result?

Reply:
Fixed